src/modules/app/route-configs/WohnenRouteConfig.js

```diff
@@ -95,7 +95,10 @@ class WohnenRouteConfig implements RouteConfig<RouteParamsType, RequiredPayloads
   getFeedbackTargetInformation = ({ payloads }) => {
     const extras = payloads.extras.data
     const extra = extras && extras.find(extra => extra.alias === WOHNEN_EXTRA)
-    return ({ alias: WOHNEN_EXTRA, title: extra && extra.title })
+    if (!extra) {
```

Comment:
Same as above, should probably return null.

Reply:
Still relevant afaics

Reply:
I did change it (https://github.com/Integreat/integreat-webapp/pull/349/files#diff-a562954fb2183a4a70967c55f860d42cR100)